Games/Retro_ML.SuperMarioBros/Neural/Scoring/TimeTakenScoreFactor.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Retro_ML.Game;
+using Retro_ML.Neural.Scoring;
+using Retro_ML.Configuration.FieldInformation;
+
+namespace Retro_ML.SuperMarioBros.Neural.Scoring
+{
+    internal class TimeTakenScoreFactor : IScoreFactor
+    {
+        public const string MAXIMUM_LEVEL_TIME = "Maximum Level Time";
+
+        private bool shouldStop = false;
+        private double currScore;
+        private int levelFrames = 0;
+
+        public TimeTakenScoreFactor()
+        {
+            ExtraFields = new ExtraField[]
+            {
+                new(MAXIMUM_LEVEL_TIME, 240)
+            };
+        }
+        public FieldInfo[] Fields => Array.Empty<FieldInfo>();
```

Comment:
Use a DoubleFieldInfo for MaximumLevelTime, don't forget to update Clone() and Update() after